node/packages/aws-lambda-otel-extension/external/otel-extension-external/otel-payloads.js

```diff
@@ -111,8 +111,7 @@ const createResourceAttributes = (fun) =>
     }
   });
 
-const createLogPayload = (eventData, logs) => {
-  const spanData = eventData.span;
+const createAttributes = (eventData) => {
```

Comment:
Can we name it `requestData` here as well?

Reply:
Ahh yeah good call 👍